nibabel/streamlines/array_sequence.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, arr_seq, common_shape, dtype):
         self.common_shape = common_shape
         n_in_row = reduce(mul, common_shape, 1)
         bytes_per_row = n_in_row * dtype.itemsize
-        self.rows_per_buf = bytes_per_row / self.bytes_per_buf
+        self.rows_per_buf = int(np.ceil(self.bytes_per_buf / bytes_per_row))
```

Comment:
Do you want `ceil` or `floor` here?

Reply:
`ceil` because I want the minimal value to be 1 for `rows_per_buf`

Reply:
But I guess you will have too many rows, oftentimes.  How about:

```
self.rows_per_buf = max(1, int(self.bytes_per_buf / bytes_per_row))
```

Reply:
Thanks, I think your way is more explicit, I will do that